typings.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'react-media' {
+  interface MediaProps {
+    query: string;
+    defaultMatches?: boolean;
+    children: ((matches: boolean) => JSX.Element) | JSX.Element
+  }
```

Comment:
we're also missing the optional `render` prop:
`render?: () => React.ReactNode;`